CONTRIBUTING.md

```diff
@@ -0,0 +1,22 @@
+# Contributing to this repo
+
+First, thank you for your interest in contributing to this project! Before you pick up your first issue and start
+changing code, please:
+ 
+1. review all documentation for the module you're interested in,  
+1. Look through the [issues for this repo,](https://github.com/filecoin-project/go-fil-components/issues)
+1. Contact one of the maintainers for guidance.
```

Comment:
A new reader probably won't know who maintainers are. How about "Write up a brief proposal in an issue."?